This PR proposes changes that would break the pipeline definition:
```
component-cli-multiarch: Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/concourse/replicator.py", line 140, in render
    definition_descriptor = self._render(definition_descriptor)
  File "/usr/lib/python3.10/site-packages/concourse/replicator.py", line 184, in _render
    'definition': factory.create_pipeline_definition(),
  File "/usr/lib/python3.10/site-packages/concourse/factory.py", line 108, in create_pipeline_definition
    validator.validate()
  File "/usr/lib/python3.10/site-packages/concourse/validator.py", line 29, in validate
    self._validate_variant(variant)
  File "/usr/lib/python3.10/site-packages/concourse/validator.py", line 34, in _validate_variant
    self._validate_element(variant)
  File "/usr/lib/python3.10/site-packages/concourse/validator.py", line 38, in _validate_element
    self._validate_element(child, parents | {element})
  File "/usr/lib/python3.10/site-packages/concourse/validator.py", line 39, in _validate_element
    element.validate()
  File "/usr/lib/python3.10/site-packages/concourse/model/traits/publish.py", line 330, in validate
    raise ModelValidationError(
model.base.ModelValidationError: must not specify platforms unless using docker-buildx as oci-builder

```

--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -27,6 +27,10 @@ component-cli:
       traits:
         draft_release: ~
         publish:
+          platforms:
+          - 'linux/x86_64'
+          - 'linux/arm64'
+          oci-builder: 'docker-buildx'
           dockerimages:
             component-cli:
               registry: 'gcr-readwrite'
@@ -48,6 +52,10 @@ component-cli:
         release:
           nextversion: 'bump_minor'
         publish:
+          platforms:
+          - 'linux/x86_64'
+          - 'linux/arm64'
+          oci-builder: 'docker-buildx'
           dockerimages:
             component-cli:
               registry: 'gcr-readwrite'

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,9 +8,10 @@ FROM eu.gcr.io/gardener-project/3rd/golang:1.18.4 AS builder
 WORKDIR /go/src/github.com/gardener/component-cli
 COPY . .
 
+ARG TARGETARCH
 ARG EFFECTIVE_VERSION
 
-RUN make install EFFECTIVE_VERSION=$EFFECTIVE_VERSION
+RUN GOARCH=$TARGETARCH make install EFFECTIVE_VERSION=$EFFECTIVE_VERSION
 
 #### BASE ####
 FROM gcr.io/distroless/static-debian11:nonroot AS base

